### PR TITLE
SVGO single files and some instructions

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ All images and other static assets you wish to include should be placed inside t
 
 Files placed in `static/` will be accessible in the following way: `static/img/image.png` will be accessible at `img/image.png`.
 
+For new SVG images execute the `yarn svgo path/to/image.svg` (or `yarn svgo-all` if you want to normalize them all) before adding them to the repo.
+
```
Comment (lipis) [inline website/README.md]: If anyone can help me with words here would be great :)

File: website/package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
     "build": "docusaurus-build",
     "publish-gh-pages": "docusaurus-publish",
     "start": "docusaurus-start",
-    "svgo": "find static | grep '\\.svg$' | xargs -Iz -n 1 svgo --pretty --indent=2 --config=svgo.yml z"
+    "svgo": "svgo --pretty --indent=2 --config=svgo.yml",
+    "svgo-all": "find static | grep '\\.svg$' | xargs -Iz -n 1 svgo --pretty --indent=2 --config=svgo.yml z"
```
Comment (j-f1) [inline website/package.json]: `s/svgo --pretty --indent=2 --config=svgo.yml/yarn svgo/`?

File: website/README.md (hunk 1 above)
Comment (j-f1) [reply website/README.md]: > Please run <code>yarn svgo *path/to/image.svg*</code> before committing a new SVG to the repository.

How about that?

File: website/README.md [hunk 3]
```diff
@@ -48,6 +48,8 @@ All images and other static assets you wish to include should be placed inside t
 
 Files placed in `static/` will be accessible in the following way: `static/img/image.png` will be accessible at `img/image.png`.
 
+Please run <code>yarn svgo *path/to/image.svg*</code> before committing a new SVG to the repository.
```
Comment (lipis) [inline website/README.md]: @j-f1 `_italic_` (according to Prettier ^^)

**Comment by lipis** [comment]
is that OK?

**Comment by lipis** [comment]
what can I do with the CI?

**Comment by j-f1** [comment]
Seems like CodeCov is just unhappy. Should be fine to merge.

**Pull request merged**